src/App/mermaidData/databaseSwitchboard/ImageClassificationMixin.js

```diff
@@ -0,0 +1,22 @@
+import axios from '../../../library/axiosRetry'
+import { getAuthorizationHeaders } from '../../../library/getAuthorizationHeaders'
+
+const ImageClassificationMixin = (Base) =>
+  class extends Base {
+    getAnnotationsForImage = async function getAnnotationsForImage(projectId, imageId) {
+      if (!imageId || !projectId) {
+        Promise.reject(this._operationMissingParameterError)
+      }
+
+      return this._isOnlineAuthenticatedAndReady
+        ? axios
+            .get(
+              `${this._apiBaseUrl}/projects/${projectId}/classification/images/${imageId}`,
+              await getAuthorizationHeaders(this._getAccessToken),
+            )
+            .then((apiResults) => apiResults.data)
```

Comment:
could add a catch here to handle errors from the axios call (in addition to the promise rejection)

Reply:
ah yeah the toast in the component works well